contracts/protocol/bases/OfferBase.sol

```diff
@@ -193,6 +195,19 @@ contract OfferBase is ProtocolBase, IBosonOfferEvents {
                             (feeAmount * fees.buyerEscalationDepositPercentage) /
                             HUNDRED_PERCENT;
                         disputeResolutionTerms.mutualizerAddress = _drParameters.mutualizerAddress;
+
+                        // Validate mutualizer interface if address is not zero (non-zero means external mutualizer)
+                        if (_drParameters.mutualizerAddress != address(0)) {
+                            try
```

Comment:
I don't know if you considered what happens if `IERC165(_drParameters.mutualizerAddress).supportsInterface` returns something that cannot be decoded into a bool.  

I suppose it's still ok - the tx would fail (without a reason), but I don't see there's any attack there. Just the caller does not get the info why it failed. But maybe give antorher thought if you haven't already.